src/Microsoft.AspNetCore.Mvc.ApiExplorer/ApiResponseTypeProvider.cs

```diff
@@ -228,5 +228,29 @@ private static bool IsClientError(int statusCode)
         {
             return statusCode >= 400 && statusCode < 500;
         }
+
+        private static bool HasSignificantMetadataProvider(IReadOnlyList<IApiResponseMetadataProvider> providers)
+        {
+            // When determining if we need to apply conventions, we need to determine if a metadata provider exists
+            // that provides metadata other than the type or status code.
+            for (var i = 0; i < providers.Count; i++)
+            {
+                var provider = providers[i];
+                
+                // [ProducesResponseType(400)] overrides conventions.
+                if (provider.StatusCode != StatusCodes.Status200OK)
+                {
+                    return true;
+                }
```

Comment:
We wouldn't `[ProducesResponseType(200)]` override the convention? I think it might make more sense to special case `[Produces("...")] as the thing that doesn't.

Reply:
Ahh. I'll just special case the attribute and add a test to cover this.